grpc/src/main/java/io/stargate/grpc/codec/cql/BytesCodec.java

```diff
@@ -0,0 +1,30 @@
+package io.stargate.grpc.codec.cql;
+
+import com.google.protobuf.ByteString;
+import io.grpc.Status;
+import io.grpc.StatusException;
+import io.stargate.db.schema.Column;
+import io.stargate.proto.QueryOuterClass;
+import io.stargate.proto.QueryOuterClass.Value;
+import io.stargate.proto.QueryOuterClass.Value.InnerCase;
+import java.nio.ByteBuffer;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+public class BytesCodec implements ValueCodec {
+  @Nullable
+  @Override
+  public ByteBuffer encode(@Nullable QueryOuterClass.Value value, @NotNull Column.ColumnType type)
```

Comment:
So non of the `ValueCodec` classes are handling cases when `value ` is `null`, which is quite strange considering that there is an explicit `@Nullable` on the param..